YCloudYUSA/y_lb_demo_content
# [DS-1042][DS-1097] Add a new demo homepage for Y LB components

Jira issues: 
- [DS-1042](https://yusa.atlassian.net/browse/DS-1042)
- [DS-1097](https://yusa.atlassian.net/browse/DS-1097)

How to check:
 - go to admin/modules
 - enable the Y Layout Builder - Init Demo Pages module
 - go to /admin/structure/migrate/manage/y_lb_init_demo_content/migrations
 - run this migration /admin/structure/migrate/manage/y_lb_init_demo_content/migrations/y_lb_init_demo_media_image/execute
 - run this migration /admin/structure/migrate/manage/y_lb_init_demo_content/migrations/y_lb_init_demo_homepage/execute
 - go to /init-demo-home-page
 - verify that page looks like on design
